src/components/admin/Checkbox/Toggle.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+import * as TW from "./Toggle.tailwind";
+
+export interface CheckboxProps {
+  label: string;
+  checked?: boolean;
+  onChange?: () => void;
+}
+
+export const Toggle: React.FC<CheckboxProps> = ({
+  label,
+  checked = false,
+  onChange,
+}) => {
+  const buttonClassName = checked
+    ? TW.checkedButtonClass
+    : TW.uncheckedButtonClass;
+  const labelClassName = checked
+    ? TW.checkedLabelClass
+    : TW.uncheckedLabelClass;
+  return (
+    // <div className="flex justify-center">
+    //   <div className="form-check form-switch">
+    //     <input checked onChange={onChange} className="form-check-input appearance-none w-9 -ml-10 rounded-full float-left h-5 align-top bg-white bg-no-repeat bg-contain bg-gray-300 focus:outline-none cursor-pointer shadow-sm" type="checkbox" role="switch" id="flexSwitchCheckDefault" />
+    //     <label className="form-check-label inline-block text-gray-800" htmlFor="flexSwitchCheckDefault">{label}</label>
+    //   </div>
+    // </div>
```

Comment:
Commenting code leads to maintenance issues. There are ways around this, noted in https://github.com/sparkletown/sparkle/blob/redesign-2021/docs/code-style.md#maybe-could-be-useful-later-dead-code

Reply:
I agree this was forgotten. I'm highly against commented code.